src/components/ApartmentItem/ApartmentItem.tsx

```diff
@@ -141,7 +143,11 @@ const ApartmentItem = (props: ApartmentItemProps) => {
                 <Text className={rentClass}>${rent}<Text className="text-sm font-normal">/month</Text></Text>
               </View>
             </View>
+            {showScore ?
```

Comment:
This is good but a better way would just be to do 
```
{showScore && <MatchPercentageBar percentage={Number(match)} fill="#f5f5f5" isSkeletonLoading={isSkeletonLoading} />
```
Instead of using a ternary operatory